stripe/src/main/java/com/stripe/android/PaymentAuthConfig.java

```diff
@@ -71,7 +69,7 @@ private Stripe3ds2Config(@NonNull Builder builder) {
 
         public static final class Builder {
             private int mTimeout = DEFAULT_TIMEOUT;
-            private UiCustomization mUiCustomization = new StripeUiCustomization();
+            private ThreeDS2UiCustomization mUiCustomization = new ThreeDS2UiCustomization();
```

Comment:
can you also make the 3ds dependency "implementation"?

Reply:
Done!